docs/EN_US/ECLLanguageReference/ECLR_mods/Recrd-Index.xml

```diff
@@ -403,6 +402,28 @@ BUILD(VehicleKey3);
             compression on the payload. The resulting index can be smaller
             than using lz4.</entry>
           </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4s'</emphasis> </emphasis></entry>
```

Comment:
Is there any way to common up this text - otherwise you will have to apply the same edits to both.